docs/static/v1/ce-helpers.js

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
We might need to update the readme or contributing docs with some info about these files

Reply:
I added a section in `README.md` describing how `index.html` pages relate with the deployed Github Pages files, and how those files are versioned.